suite-native/atoms/src/Sheet/useBottomSheetAnimation.ts

```diff
@@ -57,7 +58,11 @@ export const useBottomSheetAnimation = ({
     );
 
     const animatedSheetWrapperStyle = useAnimatedStyle(() => ({
-        top: interpolate(translatePanY.value, [-1, 0, 1], [0, 0, 1]),
+        transform: [
+            {
+                translateY: translatePanY.value - keyboard.height.value,
+            },
```

Comment:
How come we need this? I thought that keyboard avoiding view in Screen component should be enough.

Reply:
KeyboardAvoidingView does not work with Animated.View.

Reply:
nice solution